sky/spot/spot_state.py

```diff
@@ -100,19 +103,22 @@ class SpotStatus(enum.Enum):
     # Terminal statuses
     # SUCCEEDED: The job is finished successfully.
     SUCCEEDED = 'SUCCEEDED'
+    # CANCELLED: The job is cancelled by the user.
+    CANCELLED = 'CANCELLED'
     # FAILED: The job is finished with failure from the user's program.
     FAILED = 'FAILED'
     # FAILED_SETUP: The job is finished with failure from the user's setup
     # script.
     FAILED_SETUP = 'FAILED_SETUP'
+    # FAILED_CONFIG: The job is finished with failure because of invalid
```

Comment:
Do we need to update https://github.com/skypilot-org/skypilot/pull/1655/files#diff-5cca3ac210864f403e7faf92dd687a546790b752c4128d3ff23cdfce9290ab0dR123-R125 and other places to catch FAILED_CONFIG? If so, we should make those places into a func call that returns a tuple of failure statuses. Otherwise adding a new failure value here means it's easy to miss those places.

Reply:
Sorry, it seems the link is not relevant to the status?